.github/VERSIONING_POLICY.md

```diff
@@ -1,29 +1,34 @@
 
-##Semantic Versioning
+## Semantic Versioning
 The DNN Community adopted the current semantic version policy in July of 2018.  Releases before this date may follow different standards.
 
 ### Major Releases (Ex 10.0.0)
 A major release is as the name implies, a release with major changes.  These changes might include new features, breaking changes, or other larger changes.  Each major release will come with release notes that outline the nature of any known breaking changes.
 
-Major releases are also the time that platform requirements might be changed, such as requiring a new edition of SQL Server or otherwise.
+Major releases are also the time that platform requirements might be changed, such as requiring a new edition of SQL Server, .NET Framework, or otherwise.
 
 ### Minor Releases (Ex 10.1, 10.2, 10.x)
-A minor might contain smaller new features and enhancements, but will not introduce any breaking API changes, nor will it change the requirements of the hosting environment or platform to run the application.  
+A minor release might contain smaller new features and enhancements, but will not introduce any known breaking API changes, nor will it change the requirements of the hosting environment or platform to run the application.  
+
+It is possible that minor breaking changes and Javascript library updates are included in minor releases.
 
 ### Revision Releases (Ex 10.1.1, 10.1.2, 10.1.x)
-These releases are created primarily to contain hot-fix style improvements from prior releases.  Any bugs or security issues identified, or missing UI/UX features from a Minor/Major release might be added to a revision release.  Similar to a Minor release a Revision release will not contain any known breaking changes.
+These releases are created primarily to contain hot-fix style improvements from prior releases.  Any bugs or security issues identified, or missing UI/UX features from a Minor/Major release might be added to a revision release.  Similar to a Minor release a Revision release will not contain any known breaking changes API.
+
+## API Deprecation Policy (Updated September 2020)
+The DNN Platform project is in a state of transition, continuing to modernize the API and remove existing technology debt.  To this point, it will be necessary for the project to remove/restructuree many public API's.  This will be done methodically, allowing developers to transition away from the older code with time to properly respond to change.
+
+Any API method to be removed will be flagged as deprecated in a release, major, minor or revision, and will be identified to be removed by a specific version.  This will be done using a C# `[Obsolete]` annotation with a comment similar to the following "Deprecated in x.x.x.  Scheduled for removal in vy.0.0, use ____ instead".  The version number of "y" in this example must be 1 major versions ahead of the version in which the notice was added.
```

Comment:
```suggestion
Any API method to be removed will be flagged as deprecated in a release, major, minor or revision, and will be identified to be removed by a specific version.  This will be done using a C# `[Obsolete]` attribute with a comment similar to the following "Deprecated in x.x.x.  Scheduled for removal in vy.0.0, use ____ instead".  The version number of "y" in this example must be 1 major versions ahead of the version in which the notice was added.
```